include/VecCore/Backend/Libmvec.h

```diff
@@ -0,0 +1,131 @@
+#define FORCE_INLINE inline __attribute__((always_inline))
+
+#ifndef VECCORE_ENABLE_LIBMVEC
+#define VECCORE_ENABLE_LIBMVEC
+
+#include <math.h>
+
+template <typename T>
+FORCE_INLINE
+T *Frexp(const T *x, const T *exp)
+{
+  int N = *(&x + 1) - x; 
+  T *result = new T[N];
```

Comment:
Hi Arif, this will allocate memory on each call, which is makes it much slower than the speedup we would gain from vectorization. Also, this function cannot accept `T` from VecCore backends. Please take a look at the function `Cbrt` in `include/VecCore/VecMath.h` for a reference implementation that we can use for other functions. The return type cannot be a pointer, but has to be a backend type, like for the other functions. So in your implementations, please use `template <typename T>` for the template, and take into account that `T` will be one of `float`, `double`, `backend::<backend>::Real_v`, where `<backend>` is either `VcScalar`, `VcVector`, `VcSimdArray<N>`, `UMESimd`, and `UMESimdArray`.